hseling-web-cat-and-kittens/hseling_web_cat_and_kittens/main.py

```diff
@@ -16,6 +43,22 @@
 
 print(app.config)
 
+mysql = MySQL(app)
+squlitedb = SQLAlchemy(app)
+Login_Manager = LoginManager()
+Login_Manager.init_app(app)
+Login_Manager.login_view = 'login'
+
+app.config['SECRET_KEY'] = 'dmvndshjvbsdvjhvjvgfuy'
+# app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database.db'
```

Comment:
If you would like to persist your database, please use `/data/database.db` as a file to store data.